tests.py

```diff
@@ -1159,3 +1159,44 @@ def test_multiple_preferred():
         assert Qt.__binding__ == "PyQt4", (
             "PyQt4 should have been picked, "
             "instead got %s" % Qt.__binding__)
+
+    def test_pyqt_enums():
+        """Check some known enums/flags mappings."""
+        from Qt import QtCore
+
+        class_attributes = {  # Version agnostic flags and enums
+            "CheckState": ["Checked", "Unchecked", "PartiallyChecked"],
+            "Orientation": ["Horizontal", "Vertical"],
+        }
+        for class_name, attributes in class_attributes.items():
+            qt_class = getattr(QtCore.Qt, class_name)
+
+            values_path = "QtCore.Qt.{0}.{1}".format(class_name, "values")
+            assert hasattr(qt_class, "values"), (
+                "{0} should exist, but is missing".format(values_path))
```

Comment:
This is very hard to read

Reply:
I switched to `%s` notations in newest commits